guides/getting-started.md

```diff
@@ -153,8 +153,21 @@ If you want to use the `mat-icon` component with the official
 <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
 ```
 
-For more information on using Material Icons, check out the
-[Material Icons Guide](https://google.github.io/material-design-icons/).
+Additionally you have to import `MatIconModule` to your project.
+
+```ts
+import {MatButtonModule, MatCheckboxModule, MatIconModule} from '@angular/material';
+
+@NgModule({
+  ...
+  imports: [MatButtonModule, MatCheckboxModule, MatIconModule],
+  ...
+})
+export class PizzaPartyAppModule { }
+```
+
+For more information on using Material Icons, check out the [Material Icons Guide](https://google.github.io/material-design-icons/) 
+and the official documentation of [MatIconModule](https://material.angular.io/components/icon).
```

Comment:
This is a redirect to https://material.angular.io/components/icon/overview

Reply:
I'll change to https://material.angular.io/components/icon/api as soon as @Chan4077 answer my reply to his comment. I want to do just one commit addressing these coments.

Reply:
Also feel free to make suggestions about what he said: https://github.com/angular/material2/pull/9351#discussion_r161184190